extensions/interactions/EndExploration/directives/oppia-interactive-end-exploration.directive.ts

```diff
@@ -102,7 +102,7 @@ angular.module('oppia').directive('oppiaInteractiveEndExploration', [
                   var listOfIds = missingExpIds.join('", "');
                   ctrl.errorMessage = (
                     'Warning: exploration(s) with the IDs "' + listOfIds +
-                    '" will ' + 'not be shown as recommendations because' +
+                    '" will ' + 'not be shown as recommendations because ' +
```

Comment:
Can this be modified to `'" will not be shown as recommendations because ' +`?

Reply:
Done!